tabbie2.git/api/controllers/RoundController.php

```diff
@@ -46,7 +46,6 @@ public function actionFilter($tournament_id = null)
 			'query' => Round::find()
                 ->where(["round.tournament_id" => $tournament_id])
                 ->andWhere(["round.published" => 1])
-                ->andWhere(["round.displayed" => 1])
```

Comment:
Because the displayed boolean is switched when the round is started (run), but we want to give information about rounds before they actually start. The published boolean oversees that no unpublished rounds are given information about.